client/i18n/locales/english/intro.json

```diff
@@ -1023,7 +1023,7 @@
       "Through hands-on exercises and projects, you will learn the fundamentals of C#, including variables, data types, control structures, and object-oriented programming principles.",
       "By the end of this course, you will have gained the practical skills and knowledge needed to confidently leverage C# for building applications."
     ],
-    "note": "Note: You must complete the trophy challenge in each section below to qualify for the certification exam.",
+    "note": "Note: Each section below has a trophy associated with it that you must earn on the Microsoft Learn platform, and verify with freeCodeCamp, before you can qualify for the certification exam.",
```

Comment:
This could be more clear and not as a simple note. IDK if we can split things, but the only clear solution is being explicit in the steps:

Here are some important points:
1. Each section has a trophy associated with it, you need to earn them on MS Learn.
2. After earning each trophy, you need to verify them on freeCodeCamp.
3. Once you have done the above, you can qualify for the cert. exam.

Reply:
I played with this a bit, but couldn't quite get the list to render. I tried [the instructions here.](https://react.i18next.com/latest/trans-component#usage-with-simple-html-elements-like-less-than-br-greater-than-and-others-v10.4.0) and a few other things. Nothing really looked that great.